Sources/KeychainSwift.swift

```diff
@@ -95,13 +97,28 @@ open class KeychainSwift {
     let accessible = access?.value ?? KeychainSwiftAccessOptions.defaultOption.value
       
     let prefixedKey = keyWithPrefix(key)
+    var accessWithFlag:Any = accessible as Any
+    
+    var query: [String : Any]
+      
+    if let control = controlFlag{
```

Comment:
```suggestion
    if let control = controlFlag {
```

Reply:
Fixed in: 067bb33c34c39595b6b117e014665a5c661bfca4